### DISCUSSION TITLE
Plot stereographic projection perpendicular to [111] axe

### DISCUSSION THREAD

**Discussion by DSimonne** [pyxem/orix#470] [Q&A]
Hi,

I would like to make a figure like this:

![image](https://github.com/pyxem/orix/assets/51970962/eaf039c5-7af2-40bb-9b99-35a46a65b07b)
Perpendicular to the [111] axe.

 I saw the tutorial for the stereographic projection perpendicular to the [100] axe (https://orix.readthedocs.io/en/stable/tutorials/stereographic_projection.html) but I cannot figure out how to rotate it ?
 
Thank you,

David

**Comment by harripj** [comment]
@DSimonne At the moment the API only allows for poles for the stereographic projection at `(0, 0, 1)`, or `(0, 0, -1)`. A nice addition would be to support general pole points.

A good starting point to create the figure above is in the crystal directions tutorials and specifically in the [Diamond [111] pole figure section](https://orix.readthedocs.io/en/stable/tutorials/crystal_directions.html#A-diamond-[111]-pole-figure) which I think you can follow directly.
